src/JustSaying/AwsTools/MessageHandling/Dispatch/MessageDispatcher.cs

```diff
@@ -157,13 +159,13 @@ public class MessageDispatcher : IMessageDispatcher
             }
         }
 
-        private async Task<bool> CallMessageHandler(string queueName, Message message)
+        private async Task<bool> CallMessageHandler(string queueName, Message message, CancellationToken cancellationToken)
         {
             var messageType = message.GetType();
 
-            var handler = _handlerMap.Get(queueName, messageType);
+            var middleware = _middlewareMap.Get(queueName, messageType);
 
-            if (handler == null)
+            if (middleware == null)
             {
                 _logger.LogError(
                     "Failed to dispatch. Handler for message of type '{MessageTypeName}' not found in handler map.",
```

Comment:
Do you think this log message needs updating now that we're using middleware instead?

Reply:
Just re-read your description, I'll hold off on these things until it's ready for review